src/db/resolvers/post_resolver.js

```diff
@@ -49,9 +53,21 @@ type Post {
 
 */
 
-const postTypeDefs = `
+const typeDefs = `
   scalar Date
 
+  type Query {
+    getAllPosts: [Post]
+    getPostReplies: [Post]
+  }
+
+  type Mutation {
+    createPost: Post
+    updatePost(postId: String!): Post
+    deletePost(postId: String!): Post
+    votePost(postId: String!): Post
+  }
+
   type Post {
```

Comment:
We should separate queries from mutations

Reply:
when we votePost, are we updating the post? would it be possible to merge votePost into updatePost? or are they doing different things

Reply:
Hey - Good points.  updatePost will be edits to the body/title of the post.  VotePost will be upvote/downvote by other users

Reply:
Yeah I figured but I mean would it be possible to merge votePost into updatePost? or does it have to be kept separate